v4/feature/receipt/src/main/java/exchange/dydx/trading/feature/receipt/components/buyingpower/DydxReceiptBuyingPowerViewModel.kt

```diff
@@ -95,7 +97,7 @@ class DydxReceiptBuyingPowerViewModel @Inject constructor(
                 AmountText.ViewState(
                     localizer = localizer,
                     formatter = formatter,
-                    amount = subaccount?.buyingPower?.current,
+                    amount = subaccount.buyingPower?.current,
```

Comment:
why is current and postOrder optional on line `100` and `111` but not above?

Reply:
In Abacus, buyingPower is declared as optional for subaccount, but as non-optional for position.  We should clean it up at some point.